src/components/AlbumInfo/Details.css

```diff
@@ -1,36 +1,29 @@
-.grid-container {
-    display: grid;
-    grid-template-columns: 1fr 1fr;
-    grid-template-rows: 1fr 1fr;
-    gap: 10px;
-    height: 100vh;
+.top-left {
```

Comment:
All the css is doing in these files is padding and text alignment stuff, which can all be done directly on the components with the MUI sx property and the classes are only used for a single component and not reused throughout the site.  Let's add these styles directly to the components with sx and get rid of this css file.